tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/SdkBuildTasks/BuildStages/PostBuildTask.cs

```diff
@@ -98,6 +99,15 @@ private void GetApiMapFromProject()
             }
         }
 
+        private void VerifyPropsFile(string apiTag, SdkProjectMetaData sdkProject)
+        {
+            string propsFile = GetApiTagsPropsPath(sdkProject);
+            if(!File.ReadAllText(propsFile).Contains(apiTag))
```

Comment:
@dsgouda do a null/empty check here for apiTag.
What if the previous null check has been moved or worse removed from line 88
In case of null, line 105 will throw nullRef

Reply:
The function will be called only after ApiTag has been checked for null or empty (line 91) but will add the additional check